src/button/component.jsx

```diff
@@ -77,21 +77,29 @@ function isCreditDualEligible(props) : boolean {
     return true;
 }
 
-function isApmEligible(source, props) : boolean {
+let isDomainAllowed = memoize(() : boolean => {
 
-    let { locale } = normalizeProps(props, { locale: getBrowserLocale() });
+    let domain = getDomain().replace(/^https?:\/\//, '').replace(/^www\./, '');
 
-    if (getFundingConfig(source, 'allowedCountries', [ locale.country ]).indexOf(locale.country) === -1) {
+    if (!config.apmTestDomains.some(allowDomain => {
+        let regex = new RegExp(`[^a-zA-Z\\d\\-]*${ allowDomain }$`);  // eslint-disable-line security/detect-non-literal-regexp
```

Comment:
This is problematic. The `.` in the domain will be interpreted as "any character" when inserted into the regex.

Reply:
Thanks for pointing it out!  Replaced with escaped period.